meinberlin/apps/cms/models/storefronts.py

```diff
@@ -74,7 +74,13 @@ def num_entries(self):
 
     @cached_property
     def num_projects(self):
-        return Project.objects.all().count()
+        projects = Project.objects.all()\
+            .filter(is_draft=False, is_archived=False, is_public=True)
+        active_project_count = 0
+        for project in projects:
+            if project.active_phase or project.future_phases:
+                active_project_count += 1
+        return active_project_count
 
```

Comment:
would it be easier to just count active phases?

Reply:
Yes, a bit. But I think, we said, we also wanted to count the future ones? @CarolingerSeilchenspringer Currently we count all projects with active or future phases, that are published, not archived and not private. Is that like we want it? Or do we only want the active, not the future ones?